src/PaymentMethodCollectElement.res

```diff
@@ -349,30 +386,38 @@ let make = (~integrateError, ~logger) => {
           <React.Fragment>
             // Merchant's info
             <div
-              className="flex flex-col w-4/10 px-[50px] py-[80px]"
+              className="flex flex-col w-full h-max items-center p-[25px]
+                lg:w-4/10 lg:px-[50px] lg:py-[80px] lg:h-screen lg:items-end"
               style={backgroundColor: merchantTheme}>
               <div
-                className="flex flex-col self-end rounded-md shadow-lg min-w-80 w-full max-w-96"
-                style={backgroundColor: "#FEFEFE"}>
-                <div className="mx-[20px] mt-[20px] flex flex-row justify-between">
-                  <div className="font-bold text-[35px]">
+                className="flex flex-col text-white w-full min-w-[300px] max-w-[520px]
+                  lg:rounded-md lg:shadow-lg lg:min-w-80 lg:max-w-96 lg:bg-white lg:text-black">
+                <div
+                  className="flex flex-col-reverse
+                    lg:mx-[20px] lg:mt-[20px] lg:flex-row lg:justify-between">
+                  <div className="font-bold text-[48px] mt-[20px] lg:mt-0 lg:text-[35px]">
                     {React.string(`${currency} ${amount}`)}
                   </div>
-                  <img className="h-12 w-auto" src={merchantLogo} alt="O" />
+                  <div
+                    className="flex items-center justify-center h-[64px] w-[64px] bg-white rounded-sm">
+                    <img
+                      className="max-h-[48px] max-w-[64px] h-auto w-auto" src={merchantLogo} alt="O"
+                    />
+                  </div>
                 </div>
-                <div className="mx-[20px]">
+                <div className="lg:mx-[20px]">
                   <div className="self-center text-[20px] font-semibold">
                     {React.string("Payout from ")}
                     {React.string(merchantName)}
                   </div>
-                  <div className="flex flex-row mt-[5px]">
+                  <div className="flex flex-row lg:mt-[5px]">
                     <div className="font-semibold text-[12px]"> {React.string("Ref Id")} </div>
-                    <div className="ml-[5px] text-[12px] text-gray-800">
-                      {React.string(payoutId)}
-                    </div>
+                    <div className="ml-[5px] text-[12px]"> {React.string(payoutId)} </div>
                   </div>
                 </div>
-                <div className="mt-[10px] px-[20px] py-[5px] bg-gray-200 text-[13px] rounded-b-lg">
+                <div
+                  className="mt-[20px] px-[20px] py-[6px] bg-gray-200 text-[13px] rounded-full w-max text-black
```

Comment:
Please try to use predefined tailwind instead of using custom px whereever possible. For ex: instead of py-[6px] please use py-1.5